src/Main/CMakeLists.txt

```diff
@@ -79,6 +82,26 @@ set (SRC_CPP_LIST ../Main/Foedag.cpp
   ../Main/ProjectFile/FileLoaderOldStructure.cpp
   ../Main/ProjectFile/ProjectManagerComponentMigration.cpp
   ../MainWindow/QObjectContainer.cpp
+  ../InteractivePathAnalysis/NCriticalPathReportParser.cpp
```

Comment:
I think it should be separated from here into different module like we have for others e.g. compiler, texteditor, etc

Reply:
> I think it should be separated from here into different module like we have for others e.g. compiler, texteditor, etc
corrected https://github.com/os-fpga/FOEDAG/pull/1495/commits/9936845b9d4cfc33ae47cee552606d541469b645